force-app/main/default/lwc/assetManagementRelated/assetManagementRelated.js

```diff
@@ -0,0 +1,83 @@
+import { LightningElement, wire, track, api} from 'lwc';
+import getAssets from '@salesforce/apex/AssetManagementController.getAssetsByAccount'
+import renewCancel from '@salesforce/apex/AssetManagementController.renewCancelAsset'
+const columns = [
+    { label: 'Asset Name', fieldName: 'Name', type: 'text' },
+    { label: 'Lifecycle Start Date', fieldName: 'LifecycleStartDate',type: 'date', typeAttributes: {
+        day: "2-digit",
+        month: "2-digit",
+        year:"numeric",
+        timeZone:"UTC"
+    }},
+    { label: 'Lifecycle End Date', fieldName: 'LifecycleEndDate', type: 'date', typeAttributes: {
+        day: "2-digit",
+        month: "2-digit",
+        year:"numeric",
+        timeZone:"UTC"
+    }},
+    { label: 'Renewal Term Unit', fieldName: 'RenewalTermUnit', type: 'text'},
+    { label: 'Renewal Term', fieldName: 'RenewalTerm', type: 'Number'},
+];
+
+
+
+export default class AssetManagement extends LightningElement {
+    activateButtons = true;
```

Comment:
I think a lot of your properties need better names. `activateButtons` sounds like a method name, an action you take. Not a boolean flag. Typically booleans are prepended with `is` or something similar, for example.